src/Extensions/Subsites/IndexConfigurationExtension.php

```diff
@@ -4,23 +4,63 @@
 
 use SilverStripe\Core\Extension;
 use SilverStripe\SearchService\DataObject\DataObjectDocument;
+use SilverStripe\SearchService\Interfaces\DataObjectDocumentInterface;
 use SilverStripe\SearchService\Interfaces\DocumentInterface;
 
 class IndexConfigurationExtension extends Extension
 {
 
     public function updateIndexesForDocument(DocumentInterface $doc, array &$indexes): void
     {
-        $docSubsiteId = null;
+        // Skip if document object does not implement DataObject interface
+        if (!$doc instanceof DataObjectDocumentInterface) {
+            return;
+        }
+
+        $docSubsiteId = 0;
 
         if ($doc instanceof DataObjectDocument) {
```

Comment:
Do we need this check still if we have the `instanceof DataObjectDocumentInterface` check above?